feature/policy_forwarding/encapsulation/otg_tests/static_encap_gue_ipv4/README.md

```diff
@@ -0,0 +1,329 @@
+
+6.  DUT is configured with an IPv4 GUE tunnel with destination
+    IPv4-DST-GUE without any explicit tunnel Type of Service (ToS) or
+    Time to Live (TTL) values. The next hop group will be used for
+    GUE encapsulation.
+
+7.  DUT is configured with the following static routes:
+
+    -   To PNH-IPv6, next hop is the statically configured IPv4 GUE tunnel.
+    -   To IPv4-DST-GUE, next hop is ATE:LAG1:IPv4.
+    -   To IPv4-DST-GUE, next hop is ATE:LAG2:IPv4.
+
+
+### PF-1.3.1: IPv4 traffic GUE encapsulation without explicit ToS/TTL configuration on tunnel
+
+ATE action:
+
+*   Generate 12000000 **IPv4 packets** from ATE:Port1 to random IP addresses in
+    IPv4-DST-NET ensuring that there are at least 65000 different 5-tuple flows.
+    *   Use 512 bytes frame size.
+    *   Set ToS value of *0x80* for all packets.
+    *   Set TTL of all packets to *10*.
+
+Verify:
+
+*   The packet count of traffic sent from ATE:Port1 should be equal to the sum
+    of all packets:
+    *   Egressing DUT:Port2, DUT:Port3, DUT:Port4 and DUT:Port5
+        by checking `out-unicast-pkts` counter.
+    *   Received on ATE:Port2, ATE:Port3, ATE:Port4 and ATE:Port5.
+*   All packets received on ATE:Port2, ATE:Port3, ATE:Port4 and ATE:Port5 are
+    GUE encapsulated.
+*   ECMP hashing works over the two LAG interfaces with a tolerance of 6%.
+*   LAG hashing works over the two singleton ports within LAG1 and LAG2 with a
+    tolerance of 6%.
+*   ToS for all GUE encapsulated packets:
+    *   GUE header ToS is **0x80**.
+    *   Inner header ToS is **0x80**.
+*   TTL for all GUE encapsulated packets:
+    *   GUE header TTL is **9**.
+    *   Inner header TTL is **9**.
+
+### PF-1.3.2: IPv6 traffic GUE encapsulation without explicit ToS/TTL configuration on tunnel
+
+*   Modify the flows in `PF-1.3.1` to use IPv6 destination IPv6-DST-NET and
+    repeat the traffic generation and validation.
+
+### PF-1.3.3: IPv4 traffic GUE encapsulation with explicit ToS configuration on tunnel
+
+DUT and ATE actions:
+
+*   Re-configure the IPv4 GUE tunnel on the DUT with ToS value *0x60*.
+*   Generate the same flows in `PF-1.3.1`.
+
+Verify:
+
+*   Repeat same verifications in `PF-1.3.1` but with the following differences
+    *   ToS for all GUE encapsulated packets:
+        *   GUE header ToS is **0x60**.
+
+### PF-1.3.4: IPv6 traffic GUE encapsulation with explicit ToS configuration on tunnel
+
+*   Modify the flows in `PF-1.3.3` to use IPv6 destination IPv6-DST-NET and
+    repeat the traffic generation and validation.
+
+### PF-1.3.5: IPv4 traffic GUE encapsulation with explicit TTL configuration on tunnel
+
+DUT and ATE actions:
+
+*   Re-configure the IPv4 GUE tunnel on the DUT without an explicit ToS value.
+*   Re-configure the IPv4 GUE tunnel on the DUT with TTL value of *20*.
+*   Generate the same flows in `PF-1.3.1`.
+
+Verify:
+
+*   Repeat same verifications in `PF-1.3.1` but with the following differences
+    *   TTL for all GUE encapsulated packets:
```

Comment:
Verify where? if it is at the ATE then it should be 19 as the DUT will decrement the TTL before forwarding.